circuit/views.py

```diff
@@ -1,7 +1,5 @@
-from django.http import HttpResponse
-from django.template import loader
+from django.views.generic import TemplateView
 
 
-def landingpage(request):
-    template = loader.get_template("landingpage.html")
-    return HttpResponse(template.render())
+class LandingPage(TemplateView):
```

Comment:
Could be done in a separate PR, but as it's necessary for this feature it would have to be merged anyways.